input/definitions/MeasureReport/StructureDefinition-countQuantity.json

```diff
@@ -0,0 +1,78 @@
+{
+  "resourceType": "StructureDefinition",
+  "id": "countQuantity",
+  "url": "http://hl7.org/fhir/StructureDefinition/countQuantity",
+  "name": "CountQuantity",
+  "title": "Count Quantity",
+  "status": "active",
+  "experimental": false,
+  "date": "2024-01-08T13:31:32-07:00",
+  "publisher": "HL7 International / Clinical Decision Support",
+  "contact": [
+    {
+      "telecom": [
+        {
+          "system": "url",
+          "value": "http://www.hl7.org/Special/committees/dss"
+        }
+      ]
+    }
+  ],
+  "description": "Size of the population as a quantity.",
+  "purpose": "To provide support for reporting composite measures that may result in decimal-valued counts for populations.",
+  "fhirVersion": "4.0.1",
+  "baseDefinition": "http://hl7.org/fhir/StructureDefinition/Extension",
+  "derivation": "constraint",
+  "mapping": [
+    {
+      "identity": "rim",
+      "uri": "http://hl7.org/v3",
+      "name": "RIM Mapping"
+    }
+  ],
+  "kind": "complex-type",
+  "abstract": false,
+  "context": [
+    {
+      "type": "element",
+      "expression": "MeasureReport.group.population.count"
+    },
+    {
+      "type": "element",
+      "expression": "MeasureReport.evaluatedResource"
+    }
+  ],
+  "type": "Extension",
+  "differential": {
+    "element": [
+      {
+        "id": "Extension",
+        "path": "Extension",
+        "short": "Size of the population as a quantity.",
+        "definition": "A decimal representation of the size of the population.",
```

Comment:
This is confusing because the short says it's a quantity, but the definition says it's a decimal. Consider:
"A Quantity representing the size of the population. This is important to support the case when a population value may result in a decimal when reporting composite measures"